setup.py

```diff
@@ -16,7 +16,7 @@
 ]
 
 test_requirements = ["pytest", "pytest-subtests", "serialbox"]
-ndsl_requirements = ["ndsl @ git+https://github.com/NOAA-GFDL/NDSL.git@2024.03.01"]
```

Comment:
That's why we would need a release out of `main` of `pyFV3` so we can have a version that does not depend on the `develop` branch